prow/cmd/deck/static/common/common.ts

```diff
@@ -169,13 +169,16 @@ export namespace tooltip {
 }
 
 export namespace icon {
-  export function create(iconString: string, tip: string = ""): HTMLAnchorElement {
+  export function create(iconString: string, tip: string = "", onClick: string = ""): HTMLAnchorElement {
     const i = document.createElement("i");
     i.classList.add("icon-button", "material-icons");
     i.innerHTML = iconString;
     if (tip !== "") {
        i.title = tip;
     }
+    if (onClick !== "") {
+      i.setAttribute("onClick", onClick);
```

Comment:
Instead of this, attach a function directly:

```suggestion
      i.addEventListener("click", onClick);
```